docs/reference/analysis/tokenfilters/delimited-payload-tokenfilter.asciidoc

```diff
@@ -1,21 +1,331 @@
+      "end_offset": 14,
+      "type": "word",
+      "position": 1
+    },
+    {
+      "token": "fox",
+      "start_offset": 15,
+      "end_offset": 20,
+      "type": "word",
+      "position": 2
+    },
+    {
+      "token": "is",
+      "start_offset": 21,
+      "end_offset": 25,
+      "type": "word",
+      "position": 3
+    },
+    {
+      "token": "quick",
+      "start_offset": 26,
+      "end_offset": 34,
+      "type": "word",
+      "position": 4
+    }
+  ]
+}
+--------------------------------------------------
+/////////////////////
+
+[[analysis-delimited-payload-tokenfilter-analyzer-ex]]
+==== Add to an analyzer
+
+The following <<indices-create-index,create index API>> request uses the
+`delimited-payload` filter to configure a new <<analysis-custom-analyzer,custom
+analyzer>>.
+
+[source,console]
+--------------------------------------------------
+PUT delimited_payload
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "whitespace_delimited_payload": {
+          "tokenizer": "whitespace",
+          "filter": [ "delimited_payload" ]
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+
+[[analysis-delimited-payload-tokenfilter-configure-parms]]
+==== Configurable parameters
+
+`delimiter`::
+(Optional, string)
+Character used to separate tokens from payloads. Defaults to `|`. 
+
+`encoding`::
++
+--
+(Optional, string)
+Datatype for the stored payload. Valid values are:
+
+`float`:::
+(Default) Float
+
+`identity`:::
+Characters
+
+`int`:::
+Integer
+--
+
+[[analysis-delimited-payload-tokenfilter-customize]]
+==== Customize and add to an analyzer
+
+To customize the `delimited_payload` filter, duplicate it to create the basis
+for a new custom token filter. You can modify the filter using its configurable
+parameters.
+
+For example, the following <<indices-create-index,create index API>> request
+uses a custom `delimited_payload` filter to configure a new
+<<analysis-custom-analyzer,custom analyzer>>. The custom `delimited_payload`
+filter uses the `+` delimiter to separate tokens from payloads. Payloads are
+encoded as integers.
+
+[source,console]
+--------------------------------------------------
+PUT delimited_payload_example
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "whitespace_plus_delimited": {
+          "tokenizer": "whitespace",
+          "filter": [ "plus_delimited" ]
+        }
+      },
+      "filter": {
+        "plus_delimited": {
+          "type": "delimited_payload",
+          "delimiter": "+",
+          "encoding": "int"
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+
+[[analysis-delimited-payload-tokenfilter-return-stored-payloads]]
+==== Return stored payloads
+
+Use the <<indices-create-index,create index API>> to create an index that:
+
+* Includes a field that stores payloads. For this field, set the
+  <<mapping-store,`store`>> mapping parameter to `true` and the
```

Comment:
I am not sure why `store` : `true` is necessary? 

Includes a field that stores payloads -> I would reformulate it to something like "stores term vectors with payloads", as it not a usual indexed field with payloads.

Reply:
It's not. I've rephrased this bullet to "Includes a field that stores term vectors with payloads." as you suggested. Thanks!